releaser/make_release.py

```diff
@@ -409,7 +420,8 @@ def run_steps(config, steps, steps_funcs):
             step_func(config)
 
 
-def make_release(local_repository, package_name, module_name, release_name='dev', steps=':', branch='master',
-                 src_documentation=None, tmp_dir=None):
-    config = set_config(local_repository, package_name, module_name, release_name, branch, src_documentation, tmp_dir)
+def make_release(local_repository, project_name, package_name, module_name, release_name='dev', steps=':',
```

Comment:
Don't forget to update the "make_release.py" script of all projects accordingly.